proxy/hdrs/HdrToken.cc

```diff
@@ -209,6 +210,8 @@ static HdrTokenFieldInfo _hdrtoken_strs_field_initializers[] = {
   {"Proxy-Authorization", MIME_SLOTID_NONE, MIME_PRESENCE_PROXY_AUTHORIZATION, (HTIF_HOPBYHOP | HTIF_PROXYAUTH)},
   {"Proxy-Connection", MIME_SLOTID_PROXY_CONNECTION, MIME_PRESENCE_PROXY_CONNECTION, (HTIF_COMMAS | HTIF_MULTVALS | HTIF_HOPBYHOP)},
   {"Public", MIME_SLOTID_NONE, MIME_PRESENCE_PUBLIC, (HTIF_COMMAS | HTIF_MULTVALS)},
+  {"Public-Key-Pins", MIME_SLOTID_NONE, MIME_PRESENCE_NONE, HTIF_NONE},
```

Comment:
@SolidWallOfCode Should I add those line end of this array? or just remove?
Actually I'm not strongly want to add WKS strings, I just followed Strict-Transport-Security header for HSTS.